CHANGELOG.unreleased.md

```diff
@@ -7,10 +7,14 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [VM] Rescan ISO SRs button is also avaible on the console view
```

Comment:
```suggestion
- [VM/console] Ability to rescan ISO SRs 
```

Reply:
Add PR link.

Reply:
Does this PR actually make it available on the console?

Reply:
This PR no. But this PR is the continuity of [this commit](https://github.com/vatesfr/xen-orchestra/commit/d7940292d048411fa209e792c5c3c2c6d70e8312). (Commit pushed directly on the master branch by my fault).

Reply:
Ok, makes sense.